### DISCUSSION TITLE
怎么强刷el-tab里的内容？

### DISCUSSION THREAD

**Discussion by hibichann** [element-plus/element-plus#17497] [Q&A]
大屏需要轮播，el-tab的pane里，有滚动表格或者二级的其他需要时间的项目，我希望el-tab切换的时候会重新走里面组件的onmounted方法

除了点击tab时强刷el-tab-pane的key还有什么好办法？

**Comment by warmthsea** [comment]
v-if ？

**Reply by hibichann** [reply]
咋忘了这茬了
